index.php

```diff
@@ -0,0 +1,23 @@
+<?php
+    include "includes/header.php";
+
+    //write PHP here...
+	$weight = '125';
+	$age = '31';
+	$firstname = 'Brenna';
+	$lastname = 'Rosenberg';
+	function poundsToKg($weight)
+	{return $weight * (1/2.2);}

```

Comment:
nice! This formatting is a bit unorthodox - but should work. Standard way to write it is:
function poundsToKg($weight){
     //do stuff    ;
}
My only other thought here is that I like to keep my argument names unique from variable names to avoid confusion (my confusion 😄). So - I likely would have written something like: 
function poundsToKg($w) {...

Let me know if you have quesitons! You should be proud of this!